packages/web-runtime/src/App.vue

```diff
@@ -67,7 +67,23 @@ export default defineComponent({
   },
   setup() {
     const store = useStore()
+    const router = useRouter()
     useHead({ store })
+
+    const activeApp = computed(
```

Comment:
We added this to set current folder to null if app has been changed. For example this is needed when admin-settings will be entered. Otherwise the current folder will still be the last we’ve set.

Reply:
Fixed: somehow the preview app doesn't have any resource in the beginning